src/ontology/metadata/mondo.sssom.config.yml

```diff
@@ -75,6 +75,304 @@ curie_map:
   GC_ID: "http://purl.obolibrary.org/obo/mondo/mappings/unknown_prefix/GC_ID/"
   SNOMEDCT_2010_1_31: "http://purl.obolibrary.org/obo/mondo/mappings/unknown_prefix/SNOMEDCT_2010_1_31/"
 
+extended_prefix_map:
```

Comment:
## `extended_prefix_map` in `mondo.sssom.config.yml`
@matentzn Here's the new EPM. I added it within `mondo.sssom.config.yml`. I have no idea however if this could have unintended side effects, so another consideration is that I could make a separate file for the EPM entirely and load that instead.

Reply:
`HGNC__2` should be merged into HGNC to be a proper EPM!

Reply:
Yep, no worries, I'm still on it!

Reply:
Moved this work into #396, merging that, and doing a rebase here.